ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-openshift-logging.yaml

```diff
@@ -0,0 +1,91 @@
+base_images:
```

Comment:
How about rename to openshift-openshift-tests-private-release-4.12__amd64-nightly-openshift-logging-5.6.yaml?

Reply:
We may run logging 5.5 on 4.12 too?

Reply:
The job that is generated is OCP version specific, for 4.12 nightly its added to ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml .What we can do is add ci config for different OCP versions, like we have added currently for 4.12 (openshift-openshift-tests-private-release-4.12__amd64-nightly-openshift-logging.yaml) and add jobs for Logging versions that support that OCP versions. Thus we can add 5.7 to this current config. 

We can similarly add ci config for openshift-openshift-tests-private-release-4.11__amd64-nightly-openshift-logging.yaml openshift-openshift-tests-private-release-4.10__amd64-nightly-openshift-logging.yaml and add jobs for the supported Logging versions in those configs.

Reply:
For 5.5, yeah we can add that version also.